x-pack/legacy/plugins/lens/public/types.ts

```diff
@@ -6,7 +6,8 @@
 
 import { Ast } from '@kbn/interpreter/common';
 import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
-import { Query, KibanaDatatable } from 'src/plugins/data/common';
+import { Query } from 'src/plugins/data/common';
```

Comment:
Is there a reason why `Query` is imported using the `src` shortcut and `KibanaDatatable` is referencing it with a relative path?